carp.data.core/src/commonMain/kotlin/dk/cachet/carp/data/application/SyncPoint.kt

```diff
@@ -19,13 +20,38 @@ data class SyncPoint(
      */
     val synchronizedOn: Instant,
     /**
-     * The offset to be added to the sensor time stamps after having multiplied by [relativeClockSpeed].
+     * The sensor time at [synchronizedOn].
      */
     @Required
-    val utcOffset: Long = 0,
+    val sensorTimestampAtSyncPoint: Long = synchronizedOn.toEpochMicroseconds(),
     /**
-     * The value to multiply sensor time stamps by.
+     * The relative clock speed of UTC time compared to the sensor clock,
+     * calculated as the variation of UTC time divided by the variation of sensor time.
+     *
+     * E.g., if the sensor clock runs half as fast as UTC time, the relative clock speed is 2.
      */
     @Required
     val relativeClockSpeed: Double = 1.0
 )
+{
+    companion object
+    {
+        /**
+         * The default [SyncPoint] for sensors of which the clock is already synchronized to UTC time in microseconds.
+         * A synchronization conversion using this sync point is a no-op.
+         */
+        val UTC: SyncPoint = SyncPoint( Instant.fromEpochSeconds( 0 ) )
+    }
+
+
+    /**
+     * Converts [timestamp] to UTC time in microseconds using this [SyncPoint].
+     */
+    fun synchronizeTimestamp( timestamp: Long ): Long
```

Comment:
A UTC timestamp is not the same as UTC time. I think you correctly point out the documentation, and possibly method name, is misleading in that regard. This function coverts the sensor timestamp to microseconds since the Unix epoch.